packages/expo-cli/src/commands/utils/npm.ts

```diff
@@ -53,10 +53,22 @@ function applyKnownNpmPackageNameRules(name: string): string | null {
 }
 
 export async function npmViewAsync(...props: string[]): Promise<JSONValue> {
-  const cmd = ['view', ...props, '--json'];
-  const results = (await spawnAsync('npm', cmd)).stdout?.trim();
+  return npmCommandAsync(['view', ...props, '--json']);
+}
+
+export async function npmPackAsync(
+  packageName: string,
+  cwd: string | undefined = undefined,
+  ...props: string[]
+): Promise<JSONValue> {
+  return npmCommandAsync(['pack', packageName, ...props, '--json'], { cwd });
+}
+
+async function npmCommandAsync(cmd: string[], spawnOpts: SpawnOptions = {}): Promise<JSONValue> {
   const cmdString = `npm ${cmd.join(' ')}`;
   Log.debug('Run:', cmdString);
+  const results = (await spawnAsync('npm', cmd, spawnOpts)).stdout?.trim();
```

Comment:
If this function always parses as json then it should also append the `--json` flag, otherwise there's an untyped requirement to make the function work.